### PR TITLE
docs(schema): improve Pomerium documentation

### PR DESCRIPTION
**Pull request by ralgozino**
Improve schema docuemtentation for Pomerium, add some examples.

### CODE REVIEW

File: docs/schemas/ekscluster-kfd-v1alpha2.md [hunk 1]
```diff
@@ -711,6 +711,10 @@ The value of the toleration
 | [routes](#specdistributionmodulesauthpomeriumroutes)                           | `array`  | Optional |
 | [secrets](#specdistributionmodulesauthpomeriumsecrets)                         | `object` | Required |
 
+### Description
+
+Configuration for Pomerium, an indenity aware reverse proxy used for SSO.
```
Comment (Rick-1990) [inline docs/schemas/ekscluster-kfd-v1alpha2.md:716]:
```suggestion
Configuration for Pomerium, an identity-aware reverse proxy used for SSO.
```

File: docs/schemas/kfddistribution-kfd-v1alpha2.md [hunk 2]
```diff
@@ -708,6 +708,10 @@ The value of the toleration
 | [routes](#specdistributionmodulesauthpomeriumroutes)                           | `array`  | Optional |
 | [secrets](#specdistributionmodulesauthpomeriumsecrets)                         | `object` | Required |
 
+### Description
+
+Configuration for Pomerium, an indenity aware reverse proxy used for SSO.
```
Comment (Rick-1990) [inline docs/schemas/kfddistribution-kfd-v1alpha2.md:713]:
```suggestion
Configuration for Pomerium, an identity-aware reverse proxy used for SSO.
```

Comment (ralgozino) [reply docs/schemas/kfddistribution-kfd-v1alpha2.md]: good catch! fixed

**Review by Rick-1990 — APPROVED** [review]
LGTM!

**Review by alessiodionisi — APPROVED** [review]

**Pull request merged**